libzeth/tests/circuits/commitments_test.cpp

```diff
@@ -60,7 +62,7 @@ TEST(TestCOMMs, TestCOMMGadget)
     libsnark::pb_variable<FieldT> result;
     result.allocate(pb, "result");
 
-    COMM_cm_gadget<FieldT, HashT> comm_cm_gadget(pb, a_pk, rho, r, v, result);
+    COMM_cm_gadget<FieldT, Hash> comm_cm_gadget(pb, a_pk, rho, r, v, result);
```

Comment:
Should we switch `FieldT` to `Field` then?

Reply:
Switched to `Field` everywhere.